consult.el

```diff
@@ -469,7 +469,7 @@ called from the corresponding command. Note that the options depend on
 the private `consult--read' API and should not be considered as stable
 as the public API.")
 
-(defvar consult--buffer-display #'switch-to-buffer
+(defvar consult--buffer-display #'display-buffer
```

Comment:
I'm using e2wm.el and it must support `switch-to-buffer`.
However, it happens to collapse window layout in my environment.

Anyway, I believe `display-buffer` is more appropriate for this package role. So, I hope this fix be included

Reply:
No, switch-to-buffer is the better choice here. You can set switch-to-buffer-obey-display-actions or change the variable in your configuration. Please change back.